src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -679,12 +683,19 @@ def do_corpus_pruning(context, last_execution_failed, revision):
       for fuzzer in context.cross_pollinate_fuzzers
   ])
 
-  # TODO(mpherman): Add way to get tags string when tagged corpora are done.
+  # Leave tags blank if random method was used. Even if corpus has tags.
   tags = ''
+  if context.cross_pollination_method == "tagged":
+    tags = ','.join([
```

Comment:
so if a fuzz target have multiple tags, are we going to use all of them at once? I think it would be better to choose just one, because:
1) it will give us better tracking data, i.e. we should be able to see which tag works better than the other(s) for a given fuzz target
2) it will keep the number of sources for cross pollination smaller, which is also important, we shouldn't try to add too much corpora at once